imagedephi/redact.py

```diff
@@ -131,8 +131,24 @@ def _get_output_path(file_path: Path, output_dir: Path) -> Path:
     return output_dir / f"REDACTED_{file_path.name}"
 
 
+def _save_redacted_tiff(tiff_info: TiffInfo, output_path: Path, input_path: Path, overwrite: bool):
+    if overwrite or not output_path.exists():
+        if output_path.exists():
+            click.echo(f"Found existing redaction for {input_path.name}. Overwriting...")
+        tifftools.write_tiff(tiff_info, output_path, allowExisting=True)
+    else:
+        click.echo(
```

Comment:
Shouldn't we be adding `err=True` to `click.echo` calls that are errors?

Reply:
https://github.com/DigitalSlideArchive/ImageDePHI/issues/52

I created this issue to track this concern. Will look into this as a follow up.